src/Microsoft.AspNet.Razor/SourceLocation.cs

```diff
@@ -76,12 +125,13 @@ public static SourceLocation Advance(SourceLocation left, string text)
             return tracker.CurrentLocation;
         }
 
-        public static SourceLocation Add(SourceLocation left, SourceLocation right)
+        private static SourceLocation Add(SourceLocation left, SourceLocation right)
```

Comment:
hmm, makes partial sense given error checking in the `+` and `-` operators.  not clear however why these methods are separate.